docs/source/concepts/population.rst

```diff
@@ -9,8 +9,77 @@ Population Management
    :local:
    :backlinks: none
 
-Outline
--------
- - Initialization of simulants and the dependency problem.
- - Managing access with population views.
- -
+Since ``Vivarium`` is an agent-based simulation framework, managing a group of
```

Comment:
Internally, refer to vivarium as :mod:`vivarium`.  Get's us nice cross-link functionality.  

Also, vivarium itself is not agent-based (!).  There are two ways this is false:
1. vivarium itself doesn't enforce anything about the rows in the state table representing individuals.  The whole MSLT model works by treating each group as an age/sex cohort and has columns representing population counts/proportions in those groups.  This doesn't matter at all for our researchers (possibly, unless they want to do other kinds of modeling), but matters significantly for how we build the managers/engine/other abstractions in the framework itself.
2. for experts in the field there's a big distinction between agent-based modeling which focuses on individual interactions (think infectious disease dynamics) and microsimulation which models individuals but mainly cares about the impact of population heterogeneity.